components/eamxx/src/share/io/scorpio_scm_input.hpp

```diff
@@ -0,0 +1,71 @@
+#ifndef SCREAM_SCORPIO_SCM_INPUT_HPP
+#define SCREAM_SCORPIO_SCM_INPUT_HPP
+
+#include "share/grid/abstract_grid.hpp"
+
+#include <ekat/logging/ekat_logger.hpp>
+
+namespace scream
+{
+
+// Similar to AtmosphereInput, but reads in a single column from
+// a file with N columns. A few assumptions:
+//  - lat and lon variables are present in the file
+//  - fields have layout <COL [, ...]>
+class SCMInput
```

Comment:
I opted for SCMInput, since it should recall the acronym Single Column Model (SCM), used elsewhere. I thought about SCInput (Single Column Input), but SC was also very close to Surface Coupling. I'm open to change this to a better name though.